common/lib/client/realtime.js

```diff
@@ -132,11 +132,19 @@ var Realtime = (function() {
 		return channel;
 	};
 
+	/* Included to support certain niche use-cases; most users should ignore this.
+	 * Please do not use this unless you don't know what you're doing */
```

Comment:
```suggestion
	 * Please do not use this unless you know what you're doing */
```

Surely? 😉 

Same goes for the duplicate of this commentary in `rest.js`.

Reply:
lol. yes